x-pack/plugins/cases/public/components/description/index.test.tsx

```diff
@@ -139,14 +123,14 @@ describe.skip('Description', () => {
     expect(await screen.findByTestId('editable-save-markdown')).toHaveAttribute('disabled');
   });
 
-  it('should hide the edit button when the user does not have update permissions', () => {
+  it('should hide the edit button when the user does not have update permissions', async () => {
     appMockRender.render(
       <TestProviders permissions={noUpdateCasesPermissions()}>
```

Comment:
`permissions` is a parameter of `createAppMockRenderer`. Maybe we can add it there and get rid of `TestProviders`?